### ISSUE TITLE
am i softbanned?

### ISSUE THREAD

**Issue by oOBlueArrowOo** [finndev/PokeBuddy#387]
[P - 16:06:18] [Catch Pokemon] Geodude got away using a ItemPokeBall, status: CatchError
[P - 16:06:20] [UI] Updating...
[P - 16:06:22] [Catch Pokemon] NidoranFemale got away using a ItemPokeBall, status: CatchError
[P - 16:06:24] [Transferring Pokemons] No Pokemons available for transfer. Trying again later...
[P - 16:06:26] [Catch Pokemon] Zubat got away using a ItemPokeBall, status: CatchError
[P - 16:06:29] [Transferring Pokemons] No Pokemons available for transfer. Trying again later...
[P - 16:06:31] [Farm Pokestops] Populated Pokestops
[P - 16:06:31] [Catch Pokemon] Catching a lured pokemon...
[P - 16:06:31] [Catch Pokemon] Using a Berry to catch Pokemon!
[P - 16:06:31] [Catch Pokemon] Berry has been successfully used
[P - 16:06:31] [Catch Pokemon] Omanyte got away using a ItemPokeBall, status: CatchError
[P - 16:06:32] [Navigation] Walking from: 34.00935, -118.49747 to: 34.00947, -118.49734 with a distance of: 11 meters
[P - 16:06:33] [Farm Pokestops] Spinning Pokestop: 'End of Route 66'
[P - 16:06:33] [Farm Pokestops] InventoryFull -> Received XP: 50
[P - 16:06:35] [Transferring Pokemons] No Pokemons available for transfer. Trying again later...

**Comment by curtiola** [comment]
No, you got xp from a pokestop, could have something to do with the new Niantic bug, all pokemon flee chances are up

**Comment by oOBlueArrowOo** [comment]
thnx

**Comment by curtiola** [comment]
Or maybe you're out of pokeballs? The bot won't use great or ultra balls right now

**Comment by oOBlueArrowOo** [comment]
its not cathcing anything

**Comment by oOBlueArrowOo** [comment]
ohhhhhhh i just have ultra and great

**Comment by curtiola** [comment]
Check your inventory on the left, if you're out of pokeballs this could happen, by the map it might say 100 pokeballs but that includes great and ultra balls which can't be used

**Comment by curtiola** [comment]
Oh OK,  just saw your message, glad you got er figured
